### PR TITLE
Fixes #137

### LINKED ISSUE #137: Fix find

### PR DESCRIPTION
**Pull request by marirumo**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -34,14 +34,14 @@ Array.prototype._every = function(callback) {
...
-  for (let elem of array) {
+Array.prototype._find = function(callback) {
+  for (let elem of this) {
     if (callback(elem)) {
       return elem;
     }
   }
   return undefined;
-}
+};
```
Comment (eebalcarcel) [inline app.js]: Remove the semicolon ( **;** )

File: functions/find.html [hunk 2]
```diff
@@ -68,20 +69,13 @@ <h2 class="section-title">Sintaxis</h2>
           <div class="sintaxis">
             <pre>
               <code class="code syntax js">
-                function _find(array, callback) { }
+                arr.find(callback)
```
Comment (eebalcarcel) [inline functions/find.html]: Add the underscore to find

File: app.js (hunk 1 above)
Comment (eebalcarcel) [inline app.js]: Remove the semicolon ( **;** )

File: functions/find.html (hunk 2 above)
Comment (marirumo) [reply functions/find.html]: Done

**Pull request merged**